management-api/src/main/java/com/afh/skytouch/managment/queue/producers/ProductSender.java

```diff
@@ -0,0 +1,23 @@
+package com.afh.skytouch.managment.queue.producers;
+
+import com.afh.skytouch.commons.dto.GenericProduct;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductSender {
+
+    @Autowired
+    @Qualifier("createTemplate")
```

Comment:
Did you really mean to use the same template for different actions? In that case you're injecting the same bean. Avoid field injection. Check if the field should be private.